app/controllers/results_controller.rb

```diff
@@ -34,18 +34,24 @@ def show
     # Total number of "Yes" votes
```

Comment:
If you don't mind, could you also please fix the comments from these lines 31, 34, 41 to talk about `answers` instead of `votes` 🙏🏼 .

Reply:
Done! :D